_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Bitcoin Optech Newsletter #69'
+permalink: /en/newsletters/2019/10/23/
+name: 2019-10-23-newsletter
+slug: 2019-10-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of C-Lightning 0.7.3,
+requests help testing a Bitcoin Core release candidate, invites
+participation in a taproot review club, highlights updates to two
+Bitcoin wallets, and describes a few notable changes to popular
+Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade C-Lightning to version 0.7.3:** FIXME: mentioned in
+  6b1b99d7deb2d7a41fb93425239cc02854647427 but not tagged or published
+  yet
+
+- **Help test Bitcoin Core 0.19.0rc1:** production users of Bitcoin Core
+  are especially encouraged to test this [latest release
+  candidate][bitcoin core 0.19.0].
+
+## News
+
+- **Taproot review club:** starting the first week of November, several Bitcoin
+  contributors will be hosting a series of weekly meetings to help guide
+  people through review of the proposed [bip-schnorr][],
+  [bip-taproot][], and [bip-tapscript][] changes.  All developers,
+  academics, and anyone else with technical experience are welcome.  The
+  minimum expected commitment is four hours a week for seven weeks, with
+  one hour each week being a group meeting and the other three hours
+  being your own independent review of the proposals.  In addition to
+  review, developers will be encouraged to optionally implement a
+  proof-of-concept that either roughly integrates schnorr or taproot
+  into existing software or which demonstrates the new or improved
+  features the proposals make possible in Bitcoin and Lightning,
+  allowing implementers to identify flaws or sub-optimal requirements in
+  the current proposals that might be missed by people who only read the
+  documentation.
+
+    The ultimate goal of the review club is to allow participants to
+    gain enough technical familiarity with the proposals to be able to
+    either vocally support the proposals, advocate for changes to the
+    proposals, or lucidly explain why the proposals shouldn't be adopted
+    into the Bitcoin consensus rules.  Changing Bitcoin is something
+    that should be done carefully, and it's in every user's interest
+    that a large number of technical reviewers examine the proposals for
+    possible flaws before they are implemented and before users are
+    asked to consider upgrading their full nodes to enforce the new
+    rules---a change that can't be safely undone for as long as the
```

Comment:
Either expand "a change that can't be safely undone for as long as the rules affect anyone's bitcoins." into its own sentence, or remove it entirely.